settings.gradle.kts

```diff
@@ -40,3 +40,4 @@ rootDir
 // so Reloaded's Dependabot doesn't try to look into Kalium's build.gradle.kts, which is inaccessible as it is a git submodule.
 // See: https://github.com/dependabot/dependabot-core/issues/7201#issuecomment-1571319655
 apply(from = "include_builds.gradle.kts")
+include(":benchmark")
```

Comment:
```suggestion
```

Not needed.
We already include all directories that contain `/build.gradle.kts` automatically in the lines 27~37